### ISSUE TITLE
随机样本对sgd算法的影响程度

### ISSUE THREAD

**Issue by jiankg** [huanghao/learning-tensorflow#9]
tf.train.shuffle_bacth解析
